cli/dcoscli/experimental/main.py

```diff
@@ -54,21 +54,33 @@ def _cmds():
             arg_keys=['--dcos-package',
                       '--package-name', '--package-version'],
             function=_add),
-
         cmds.Command(
             hierarchy=['package', 'build'],
             arg_keys=['<build-definition>', '--output-directory'],
             function=_build,
         ),
+        cmds.Command(
+            hierarchy=['experimental', 'service', 'start'],
+            arg_keys=['<package-name>', '--package-version', '--options'],
+            function=_service_start),
```

Comment:
The number of words/tokens in the command. The length of the example above is 3.